Tools/i18n/msgfmt.py

```diff
@@ -158,14 +158,19 @@ def make(filename, outfile):
             msgctxt = b''
         elif l.startswith('msgid') and not l.startswith('msgid_plural'):
             if section == STR:
-                add(msgctxt, msgid, msgstr, fuzzy)
-                msgctxt = None
                 if not msgid:
+                    # Filter out POT-Creation-Date
+                    # See issue #131852
+                    msgstr = b''.join(line for line in msgstr.splitlines(True)
+                                      if not line.startswith(b'POT-Creation-Date'))
```

Comment:
```suggestion
                                      if not line.startswith(b'POT-Creation-Date:'))
```
This is what GNU `msgfmt` does.